packagehandlers/unsupportedpackagehandler.go

```diff
@@ -10,7 +10,7 @@ type UnsupportedPackageHandler struct {
 }
 
 func (uph *UnsupportedPackageHandler) UpdateDependency(vulnDetails *utils.VulnerabilityDetails) error {
-	return errors.New("frogbot currently does not support opening a pull request that fixes vulnerabilities in " + vulnDetails.Technology.ToFormal())
+	return errors.New("frogbot currently does not support scanning vulnerabilities in " + vulnDetails.Technology.ToFormal())
```

Comment:
why did we change the error message? we need to emphasize here that we cannot perform the fixing process. This is not the scanning part and might be confusing